### DISCUSSION TITLE
Add "Edit on GitBook"

### DISCUSSION THREAD

**Discussion by igorwojda** [GitbookIO/community#343] [Feature Requests]
- I am logged into the "Edit Dashboard"
- I am viewing live docs
- I am seeing this
<img width="284" alt="image" src="https://github.com/GitbookIO/community/assets/530122/41f2cf5e-6162-41c5-b736-bc7a1b0d414b">

Please add "Edit on GitBook" item to the menu

**Comment by taszap** [comment]
Hi @igorwojda,

Thanks for your feedback, out of curiosity why would "edit on GitBook" be useful to you?
If you logged in to the editor and you are in live edits, then you can edit the content immediately by typing. 

Unless I misunderstood and you mean external contributions on the published documentation?

**Comment by igorwojda** [comment]
As an author/admin/writer, I am looking at live docs.
I spot the typo or thing that can be improved, and I want to quickly jump
to page edit (instead of opening the "editing panel" and looking for
exactly the same page) and hit the *Edit* button there.
Since I am also logged in "editing panel" and I have edit rights I would
like to be able to quickly update pages.

Hope this helps.


On Fri, 27 Oct 2023 at 22:03, Natasza ***@***.***> wrote:

> Hi @igorwojda <https://github.com/igorwojda>,
>
> Thanks for your feedback, out of curiosity why would "edit on GitBook" be
> useful to you?
> If you logged in to the editor and you are in live edits, then you can
> edit the content immediately by typing.
>
> Unless I misunderstood and you mean external contributions on the
> published documentation?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/GitbookIO/discussions/343#discussioncomment-7407211>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAEBNSXFCTD3JLVEA37YYO3YBQHS3AVCNFSM6AAAAAA5ADSQXOVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TIMBXGIYTC>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
*Igor Wojda*

**Reply by taszap** [reply]
Thanks for the clarification! Makes sense. 
I think if we were to build this it would have to be carefully considered because at the moment we do not support external contributions (outside of GitHub) so this button could lead to confusion/expectation that external users can also contribute. 
I don't think it's something we would tackle in the near future but if we do have any updates I will make sure to share them with you.

**Reply by Matt-Stringer-GGL** [reply]
Having this as an option would be quite helpful.  

For our use case where it would be useful:
- We're using visitor authentication on a site for some of our internal company documentation.
- The site provides a nice clean view of the documentation for when people are just looking things up
- We want people to be able to easily contribute edits if they see that something is not quite correct.  The potential editors are the same group as the audience for the site.
- At the moment we're manually customizing the site to add an "Edit in GitBook" button, but that only helps with navigating to the relevant space, not the individual page.

**Comment by igorwojda** [comment]
This button should be only visible to internal contributors who can
actually edit this page.

On Wed, 1 Nov 2023 at 10:28, Natasza ***@***.***> wrote:

> Thanks for the clarification! Makes sense.
> I think if we were to build this it would have to be carefully considered
> because at the moment we do not support external contributions (outside of
> GitHub) so this button could lead to confusion/expectation that external
> users can also contribute.
> I don't think it's something we would tackle in the near future but if we
> do have any updates I will make sure to share them with you.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/GitbookIO/discussions/343#discussioncomment-7443847>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAEBNSSH6XTITZ6HBRTXJK3YCII2NAVCNFSM6AAAAAA5ADSQXOVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TINBTHA2DO>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
*Igor Wojda*
